packages/api/endpoints/rules.js

```diff
@@ -137,48 +141,38 @@ async function put(req, res) {
   const { params: { name }, body } = req;
 
   const apiRule = { ...body };
-  let newRule;
+  let translatedRule;
 
   if (name !== apiRule.name) {
     return res.boom.badRequest(`Expected rule name to be '${name}', but found`
       + ` '${body.name}' in payload`);
   }
 
   try {
-    const oldRule = await ruleModel.get({ name });
+    const oldRule = await rulePgModel.get(knex, { name });
+    const oldApiRule = await translatePostgresRuleToApiRule(oldRule, knex);
+    const translatedApiRule = await translateApiRuleToPostgresRuleRaw(apiRule, knex);
```

Comment:
```suggestion
    const apiPgRule = await translateApiRuleToPostgresRuleRaw(apiRule, knex);
```

This is a nit, but changing this variable name helps me follow the logic a bit more

Reply:
[`b891209` (#2871)](https://github.com/nasa/cumulus/pull/2871/commits/b891209fba38568d9aa132371d7cb4d886730ee6)